src/actions/types.tsx

```diff
@@ -1,5 +1,5 @@
 // Salary types
-export const NEW_SALARY = 'NEW_SALARY';
+export const NEW_EMPLOYEE = 'NEW_EMPLOYEE';
```

Comment:
Instead of defining actions as `const` we typically use an enum instead, e.g.
```typescript
export Enum EmployeeActions {
  NEW_EMPLOYEE = 'NEW_EMPLOYEE' //would rename this to ADD_EMPLOYEE for clarity
  UPDATE_SALARY_SUM = 'UPDATE_SALARY_SUM'
}
```